src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,31 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $cell-size);
+  justify-content: center;
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
+
+  @media (min-width: 487px) {
+    grid-template-columns: repeat(2, $cell-size);
+  }
```

Comment:
you should make it 488px, because when i set window width 487px, there is still 2 columns, but under 488px must be only one
![image](https://user-images.githubusercontent.com/122156399/216349879-0d11d75d-c051-4b69-b802-cec52d3418f6.png)